sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 2.1.0 (Unreleased)
```

Comment:
Are we following semver strictly for this package? If so, this should be a 3.0.0 (totally ok with saying it's too much effort to do a major bump for this internal tool though)

Reply:
Yup good point, sounds good to me